ArticleTemplates/outbrainTemplate.html

```diff
@@ -1,5 +1 @@
-<div class="container container__outbrain" id="outbrain">
```

Comment:
There's a few CSS styles targeting this selector that can probably be removed `.container__outbrain`